src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerSpec.scala

```diff
@@ -1901,6 +1902,71 @@ class JsonRpcControllerSpec
     response should haveObjectResult("transactions" -> JArray(expectedTxs.toList))
   }
 
+  "request pending transactions and return valid response" should "mempool is empty" in new TestSetup {
+    (ethService.ethPendingTransactions _)
+      .expects(EthPendingTransactionsRequest())
+      .returning(Future.successful(Right(EthPendingTransactionsResponse(List()))))
+
+    val request = JsonRpcRequest(
+      "2.0",
+      "eth_pendingTransactions",
+      Some(
+        JArray(
+          List()
+        )
+      ),
+      Some(JInt(1))
+    )
+
+    val response: JsonRpcResponse = jsonRpcController.handleRequest(request).futureValue
+
+    response should haveResult(JArray(List()))
+  }
+
+  it should "mempool is empty" in new TestSetup {
```

Comment:
Mempool has transactions here

Reply:
Good catch, thank you it gave me some good hint on why tests failed after porting from qa scope to eth 👍